marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/provenance/migrateProvenance.api

```diff
@@ -0,0 +1,25 @@
+{
+  "functionName": "migrateProvenance",
+  "endpoint": "/data-hub/5/data-services/provenance/migrateProvenance.sjs",
+  "desc": "Migrates provenance to provenance format starting in DH 5.7.",
```

Comment:
Should this be DHF 5.6.0?

Reply:
No. We aren't advertising the latestProvenance flag and the default provenance won't change until 5.7.